compiler/one-cmds/tests/one-codegen_007.test

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
+
+# Copyright (c) 2024 Samsung Electronics Co., Ltd. All Rights Reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# run dummy-compile drvier with given target overwrited from the command line
+
+filename_ext="$(basename -- $0)"
+filename="${filename_ext%.*}"
+
+trap_err_onexit()
+{
+  echo "${filename_ext} FAILED"
+  rm -rf ../bin/dummy-compile
+  exit 255
+}
+
+trap trap_err_onexit ERR
+
+outputfile="sample.tvn"
+configfile="one-codegen_006.cfg"
+
+rm -rf ${outputfile}
+rm -f ${filename}.log
+
+# copy dummy-compile to bin folder
+cp dummy-compile ../bin/dummy-compile
+
+# run test
+one-codegen -C ${configfile} -T rose > ${filename}.log 2>&1
+
+rm -rf ../bin/dummy-compile
+
+if grep -q "dummy-compile with rose target" "${outputfile}"; then
```

Comment:
cfg has `sunflower` but this checks `rose` .. is this negative test?

Reply:
@seanshpark No. this test checks if the target option can be overwritten by the command line. The command line gives `rose` as a target option.

```bash
one-codegen -C ${configfile} -T rose > ${filename}.log 2>&1
```